### PR TITLE
docs: add example for: apigateway method

### PR DESCRIPTION
**Pull request by dansantf**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

Relates OR Closes #0000

### CODE REVIEW

File: templates/resources/apigateway_method.md.tmpl [hunk 1]
```diff
@@ -17,6 +17,18 @@ Creates a mock method GET
...
 
+### Basic use of API method response
+
+Basic use of API method response with only status code
+
+{{ tffile (printf "examples/resources/%s/apigateway_method_basic_example.tf" .Name)}}
+
+### Use of API method response with response parameters and response models
+
+API method repsosne using response models and response parameters
```
Comment (quixoticmonk) [inline templates/resources/apigateway_method.md.tmpl:28]: small typo . `repsosne` -> `response`

**Comment by wellsiau-aws** [comment]
@dansantf , thanks for fixing the typo, can you please re-run `make docs` to incorporate the fixes

**Review by wellsiau-aws — APPROVED** [review]
LGTM

**Comment by marcosentino** [comment]
@dansantf Thank you for the contribution!

**Pull request merged**